client/state/first-view/actions.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Internal dependencies
+ */
+import {
+	FIRST_VIEW_ENABLE,
+	FIRST_VIEW_DISABLE,
+	FIRST_VIEW_HIDE,
+	FIRST_VIEW_SHOW,
+} from 'state/action-types';
+
+export function disableView( { view } ) {
+	return {
+		type: FIRST_VIEW_DISABLE,
+		view,
+	};
+}
+
+export function enableView( { view } ) {
+	return {
+		type: FIRST_VIEW_ENABLE,
+		view,
+	};
+}
+
+export function hideView( { view } ) {
```

Comment:
When integrating this with the UI, I found that it didn't really make sense to make a separate call to disable a view. Instead, I wanted to set the enabled state at the same time I hide the view (because, I don't really want to change the enabled state when the checkbox changes -- I only want to do it when the user dismisses the first view).

So, something like `hideView( { view, enabled } )` would work nicely.